chanzuckerberg/idseq-web
# Add metadata update times to heatmap_ts

# Description

See https://jira.czi.team/browse/IDSEQ-1450 . This will change the heatmap cache key whenever there is an update to underlying metadata. 

# Tests

* Go to a heatmap
* Inspect `x-cache-key` http header
* Change metadata for a sample in rails console
* See cache key change
